app/index.html

```diff
@@ -124,6 +125,8 @@
         </div>
         <save-virtual-study :show-save-button="showSaveButton" :stats="stats"
```

Comment:
save-virtual-study should be disabled now

Reply:
I disabled it by setting iViz.vue.manage.getInstance().showSaveButton = false and it won't be displayed on page.

Reply:
I'm wondering since we are not using the save button, should we comment out the whole function. But I guess configuration should work too.